app/cisc/cisc.go

```diff
@@ -290,8 +324,66 @@ func sshSync(c *cli.Context) error {
 	log.Fatal("Not yet implemented")
 	return nil
 }
-func proposeSendVoteUpdate(cfg *identity.Identity, p *identity.Config) {
-	log.ErrFatal(cfg.ProposeSend(p))
-	log.ErrFatal(cfg.ProposeVote(true))
-	log.ErrFatal(cfg.ConfigUpdate())
+
+func followAdd(c *cli.Context) error {
+	if c.NArg() < 2 {
+		log.Fatal("Please give a group-definition, an ID, and optionally a service-name of the skipchain to follow")
+	}
+	cfg, _ := loadConfig(c)
+	group := getGroup(c)
+	idBytes, err := hex.DecodeString(c.Args().Get(1))
+	log.ErrFatal(err)
+	id := identity.ID(idBytes)
+	newID, err := identity.NewIdentityFromCothority(group.Roster, id)
+	log.ErrFatal(err)
+	if c.NArg() == 3 {
+		newID.DeviceName = c.Args().Get(2)
+	} else {
+		var err error
+		newID.DeviceName, err = os.Hostname()
+		log.ErrFatal(err)
+	}
+	cfg.Follow = append(cfg.Follow, newID)
+	cfg.writeAuthorizedKeys(c)
+	// Identity needs to exist, else saving/loading will fail. For
+	// followers it doesn't matter if the identity will be overwritten,
+	// as it is not used.
+	cfg.Identity = newID
+	return cfg.saveConfig(c)
+}
+func followDel(c *cli.Context) error {
+	if c.NArg() != 1 {
+		log.Fatal("Please give id of skipchain to unfollow")
+	}
+	cfg := loadConfigOrFail(c)
+	idBytes, err := hex.DecodeString(c.Args().First())
+	log.ErrFatal(err)
+	idDel := identity.ID(idBytes)
+	var newSlice []*identity.Identity
+	for _, id := range cfg.Follow {
+		if !bytes.Equal(id.ID, idDel) {
```

Comment:
Nitpick and because it's cool: https://github.com/golang/go/wiki/SliceTricks#filtering-without-allocating